### ISSUE TITLE
Implement Nominatum search

### ISSUE THREAD

**Issue by switzersc-usds** [usds/justice40-tool#909]
Let's do this behind a feature flag for now. 

Acceptance Criteria:
- [x] design for placement of search bar
- [ ] design for parameters / instructions / tooltip / placeholder text for search
- [x] dev puts this behind feature flag
- [ ] dev implementation of UI and search so user can search by zipcode. stretch goal is city, address, etc, pending findings on experience. search by those fields should come out of the box with the search api, but whether we _tell_ people to search by those things is yet to be determined.
- [ ] QA with the list of 30 places

This of course will not be a perfect or finished search experience. The goal of this is to get to a minimally acceptable search in preparation for beta launch.

## Dev Tasks
What will trigger the app to switch from one solution to the other?

API
- [x] understand query params and options for sending data to Nominatum API
- [x] understand various ways the return data can be sent
- [x] data-fetch client investigation, `axios` vs `swr` vs `fetch` vs `react-query`

Search UI component
- [x] display the search field
- [x] truss has a search component with intl - limited options on what to display
- [x] can trigger action on search
- [x] trigger Nominatum on search
- [x] place this behind a feature flag
- [ ] error message component (blocked on this with a strange bug)
- [x] unit tests
- [x] intl
- [ ] omb laptop issue
- [ ] search should be disabled if service is down via `/status` endpoint
- [ ] ensure tab order is correct

Map
- [x] pan map pan based on lat/long
- [x] pans based on search

**Comment by switzersc-usds** [comment]
See #895 for readout from spike

**Comment by katherinedm-usds** [comment]
@vim-usds see https://app.zenhub.com/workspaces/justice40-60993f6e05473d0010ec44e3/issues/usds/justice40-tool/805 for mockups of Search bar
